docs/migration/5_0.md

```diff
@@ -775,6 +775,10 @@ Due to i18next migration, certain set of keys have been migrated (from `_plural`
 
 - Text type icon symbols are now converted into text, meaning DOM does not render HTML symbols anymore.
 
+### json-ld
+
+- For security improvement, html sanitizer has been removed from `JsonLdScriptFactory`, now replaced by html characters escape. It impacts `JsonLdDirective` and `StructuredDataFactory`.
+
```

Comment:
Did you check if the JSON LD is rendered in SSR mode?

Reply:
it is fine on SSR, json-ld correctly rendered, tested on ssr localhost.